pages/deployments/deploying_to_kubernetes.md.erb

```diff
@@ -0,0 +1,222 @@
+# Deploying to Kubernetes
+
+<%= estimated_time "30 minutes" %>
+
+<%= toc %>
+
+This tutorial shows how to deploy to Kubernetes using separate test and deploy
+[pipelines][]. Separating pipelines like this is best practice. The test
+pipeline runs tests and push a Docker image to a registry. The deploy pipelines
+uses the `DOCKER_IMAGE` enviornment variable to create a [Kubernetes
+deployment][k8s_deployment] via `kubectl`. Then, you'll see how to link them
+together to automate deploys from master.
+
+First up, you need to add a step to your existing test pipline that pushes a
+Docker image. Also check your agents have `kubectl` access to your target
+cluster. Refer to the notes at the end of tutorial for tips on setting this
+up.
+
+<%= image "final_test_pipeline.png", width: 736, height: 199, alt: 'Final Test Pipeline' %>
+
+Let's go.
+
+## Create the Deploy Pipeline
+
+This section covers creating a new Buildkite pipeline that loads steps from
+`.buildkite/pipeline.deploy.yml`. Deploy pipelines are like any other pipeline
+except they're triggred by events other than code pushes. We'll use a [trigger
```

Comment:
'They're triggred by events other than code pushes' kind of makes it sound like they're a special inbuilt pipeline type? I reckon just pull that out, the rest of the paragraph sits fine without that middle sentence.